src/Modules/Trinity.FFI/Trinity.FFI/Agent/Storage.CellManager.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using Trinity.Storage;
+using Trinity.TSL.Lib;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Trinity.FFI.Agent
+{
+   public static partial class Storage
+    {
+        public class CellManager : IDisposable
+        {
+            /// <summary>
+            /// CacheStorageManager is to manage global cell caches, 
+            /// whose context is shared for all applications in the same process.
+            /// </summary>
+            private int moduleId;
+            public int ModuleId { get => moduleId;}
+
+            static ObjectStore<ICell> Cache = new ObjectStore<ICell> { };
+
+            
+
+            public CellManager()
+            {
+                Cache = new ObjectStore<ICell> { };
```

Comment:
do not allocate static member in a ctor.

Reply:
@yatli `IDAllocator` is to assign an ID for a cell... We talked about it that day and the automatically id assignment of Trinity's IDFactory didn't work until now.  
I thought you might have better ideas to solve automatically ID assignment, so I didn't make additional changes in .NET...

Reply:
@thautwarm got you. I'll work on distributed consistent id allocator.